api/src/main/java/com/exelaration/abstractmemery/controllers/CreateController.java

```diff
@@ -0,0 +1,39 @@
+package com.exelaration.abstractmemery.controllers;
+
+import com.exelaration.abstractmemery.domains.Caption;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.exelaration.abstractmemery.domains.Caption;
+
+@RestController
+@RequestMapping("/create")
+@CrossOrigin(origins = "http://localhost:3000")
+public class CreateController {
```

Comment:
Can we name this controller and the related path to something involving memes?  Perhaps `MemeController` and `/meme`.  The `POST` action of this would then denote creating a meme, `GET` would denote loading all memes or a particular meme, etc.